### ISSUE TITLE
[MAS1.4.3] [Visual Requirements-Create a new bot configuration] Luminosity Ratio is less than 4.5:1 for “Hide, Copy, Error message” controls

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1743]
**Actual Result:**
Luminosity Ratio is less than 4.5:1 for “Hide, Copy, Error message” controls and Learn more links 

**Expected Result:**
Luminosity Ratio should be greater than or equal to 4.5:1 for “Hide, Copy, Error message” controls and Learn more links

**Note**: Same issue repro for below path 
Resources->Transcripts and chat files->choose a different location.
Luminosity ration is less than 4.5:1 for Browse control
Please refer the attachment:MAS1 4 3_Choose different location

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Welcome tab screen gets opened
3.Navigate to the Create a new bot configuration and select it
4.Dialog gest opened
5.Navigate to the “Hide, Copy, Error message” controls and Learn more links  
6.Open 'Colour Contrast Analyser' application.
7. Verify whether Luminosity Ratio is less than 4.5:1 for “Hide, Copy, Error message” controls and Learn more links  

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of link is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Tool: Colour contrast analyser

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
![MAS1 4 3_Create new bot configuration](https://user-images.githubusercontent.com/53518994/63261870-21426280-c2a2-11e9-9383-c5cb1eb72cb2.png)
![MAS1 4 3_Choose different location](https://user-images.githubusercontent.com/53518994/63588840-1864d000-c5c5-11e9-886c-08fc1517c0b8.png)

**Comment by corinagum** [comment]
Looping @DesignPolice in - I have fixed the anchor color, but I wanted to check with you on the yellow text. (I also sent this message on Teams, if you need more context.

Currently, the text color is #B89500 on `#f4f4f4` background, but the contrast ratio isn't high enough.

![image](https://user-images.githubusercontent.com/14900841/63897694-f72c2700-c9aa-11e9-8b7f-6779f826bbb8.png)


Note, anchor text is fixed in PR #1787

**Comment by corinagum** [comment]
@DesignPolice offline conversation introduced the possibility of changing the text to grey, and adding a warning icon to indicate type of message.

**Comment by Kumar2608** [comment]
Verified in latest bot emulator Version 4.5.2-79874 and issue got fixed .Hence we are clsoing this bug.
Now Luminosity Ratio for Learn more ,Hide and copy controls  >=4.5:1
